### PR TITLE
Fix issue with broken output stream

### PR DESCRIPTION
**Pull request by victor-onofrei**

### CODE REVIEW

File: Resolve-PendingMigrations.ps1 [hunk 1]
```diff
@@ -29,9 +29,9 @@ foreach ($mailbox in $allMailboxes) {
             Set-RemoteMailbox $mailboxInfo.UserPrincipalName -EmailAddresses $mailboxInfo.EmailAddresses `
              -EmailAddressPolicyEnabled $false
         } else {
-            Write-Host "Mailbox" $mailbox "has on-premise archive"
+            Write-Warning "Mailbox $mailbox has on-premise archive"
         }
     } else {
-        Write-Host "Mailbox" $mailbox "has on-premise content"
+        Write-Warning "Mailbox $mailbox has on-premise content"
```
Comment (revolter) [inline Resolve-PendingMigrations.ps1:35]: You could enclose the variable in single quotes, so the output is clearer for the user.

Comment (victor-onofrei) [reply Resolve-PendingMigrations.ps1:35]: Do you feel it helps?

PS C:\windows\system32> Write-warning -Message "mailbox $mailbox has onprem archive"
WARNING: mailbox user@domain.com has onprem archive
PS C:\windows\system32> Write-warning -Message "mailbox '$mailbox' has onprem archive"
WARNING: mailbox 'user@domain.com' has onprem archive

Comment (revolter) [reply Resolve-PendingMigrations.ps1:35]: Oh, didn't know that that's an email address.

Comment (victor-onofrei) [reply Resolve-PendingMigrations.ps1:35]: It isn't. It's usually just the alias like
`WARNING: mailbox 'vo456457' has onprem archive`

Comment (revolter) [reply Resolve-PendingMigrations.ps1:35]: Too late 😂 It's ok though; at least it doesn't contain spaces.

Comment (victor-onofrei) [reply Resolve-PendingMigrations.ps1:35]: image

Comment (victor-onofrei) [reply Resolve-PendingMigrations.ps1:35]: image

**Pull request merged**